src/entity/Comment.ts

```diff
@@ -60,4 +69,10 @@ export class Comment {
     onDelete: 'CASCADE',
   })
   parent: Promise<Comment>;
+
+  @BeforeInsert()
+  @BeforeUpdate()
+  setContentHTML(): void {
+    this.content_html = markdown.render(this.content);
```

Comment:
I just noticed `content_html` is not camelCase. Please fix it according to our convention